fv3gfs/util/communicator.py

```diff
@@ -91,6 +80,48 @@ def wait(self):
                 Buffer.push_to_cache(transfer_buffer)
 
 
+class HaloUpdateRequestMessage:
+    """Asynchronous request object for halo updates."""
+
+    def __init__(
+        self,
+        send_requests: List[AsyncRequest],
```

Comment:
You may be able to use a looser type hint, it looks like this only needs an `Iterable[AsyncRequest]`.

Reply:
Yep